ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -291,6 +299,30 @@ def _extract_events(events: Sequence[Event]) -> ProtoSpan.TimeEvents:
     )
 
 
+OT_RESOURCE_LABEL_TO_GCP = {
+    "gce_instance": {
+        "cloud.account.id": "project_id",
+        "host.id": "instance_id",
+        "cloud.zone": "zone",
+    }
+}
+
+
+def _extract_resources(resource: Resource) -> Dict[str, str]:
```

Comment:
Will `_get_monitored_resource`  & `_extract_resources` always use the same attribute keys & values? Since these methods will grow with more complex mappings over time, it might be worth putting some effort into re-using the same mapping code across both exporters?

Reply:
They will be similar yes. The difficulty right now is because the trace and monitoring exporter are in different packages. I plan on moving them into the same package in a later PR, after which I can do a lot of generalization work. 

Tracked here https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/issues/11 but resolved for now.